activiti-cloud-services-runtime-bundle/activiti-cloud-services-events/src/main/java/org/activiti/cloud/services/events/converter/EventConverterContext.java

```diff
@@ -70,35 +77,64 @@ public ProcessEngineEvent from(ActivitiEvent activitiEvent) {
     }
 
     public static String getPrefix(ActivitiEvent activitiEvent) {
+        if (isProcessEvent(activitiEvent)) {
```

Comment:
@constantin-ciobotaru should these be else ifs? instead of all ifs only?

Reply:
'ifs' or 'elseifs' is the same because we exit with 'return'.

Reply:
same, but more clear with if else in my opinion.. it is just about style.. that's clear..

Reply:
understood that, that's why I've changed to elseif.